utils/rollDown/SequencerStaking.ts

```diff
@@ -35,6 +36,8 @@ export class SequencerStaking {
       chainName,
       stringToBN(amountToStake.toString()),
       null,
+      //@ts-ignore
+      stakeAction,
```

Comment:
ask for new types, install them, and remove the ts-ignore.

Reply:
Done